core/app/models/mno_enterprise/tenant_config.rb

```diff
@@ -156,9 +156,9 @@ def self.update_locales_list!
     # Populate the app list after fetching it from MnoHub
     # TODO: replace with a Proc.call
     def self.update_application_list!
-      available_app_nids = App.all.map(&:nid)
+      available_app_nids = App.where(active: true).map(&:nid)
```

Comment:
Should we add a default scope to App?
I don't think mnoe should ever see inactive apps?

Reply:
Or it could be done on the hub side

Reply:
Makes sense, maybe best to add it as a filter at the resource level in hub?

Reply:
Yeah, should be by default I reckon?

The inactive apps shouldn't be exposed on the API. We only need them in the DevPlatformSync job to re-enable them if they get re-activated from the DevPlatform